### PR TITLE
PPoF a11y

### PR DESCRIPTION
**Pull request by nbierdeman**
### Description

This PR adds aria-labels for the PPoF flow. Please see the video below for a demonstration of these changes.

### Demo

https://user-images.githubusercontent.com/20399044/207972334-5119488a-8067-4720-be7e-60e09b9f39db.mov

### CODE REVIEW

**Review by wsbrunson — APPROVED** [review]

**Review by jshawl — APPROVED** [review]
looks good! i think one enhancement could be creating a function that we can unit test like `getAriaLabel({instrument, fundingConfig, showPayLabel})`

**Comment by Sarkerversion2030** [comment]
https://github.com/paypal/paypal-checkout-components/pull/2048#pullrequestreview-1219982945

File: src/ui/buttons/button.jsx [hunk 1]
```diff
@@ -99,8 +99,14 @@ export function Button({ fundingSource, style, multiple, locale, env, fundingEli
...
-    
-    const labelText = typeof fundingConfig.labelText === 'function' ?  fundingConfig.labelText({ content, fundingEligibility }) : (fundingConfig.labelText || fundingSource);
+
+    let labelText;
+
+    if (!showPayLabel && instrument?.vendor && instrument.label) {
+        labelText = instrument.secondaryInstruments ? `${instrument.secondaryInstruments[0].type} & ${instrument.vendor} ${instrument.label}` : `${instrument.vendor} ${instrument.label}`;
+    } else {
+        labelText = typeof fundingConfig.labelText === 'function' ?  fundingConfig.labelText({ content, fundingEligibility }) : (fundingConfig.labelText || fundingSource);
+    }
```
[the remark this answers is not in the document]
Comment (nbierdeman) [reply src/ui/buttons/button.jsx:109]: Thanks for the feedback, @jshawl! `showPayLabel` will be falsey for any non-PPoF flow. @ravishekhar and I may rename that variable at some point, but naming things is hard. 🤣 

Want me to write a function for this then?

[the remark this answers is not in the document]
Comment (nbierdeman) [reply src/ui/buttons/button.jsx]: @jshawl, I just pushed a commit that moved the aria-label logic to a function. Please have a look and let me know if you think that's a better option. 👀

[the remark this answers is not in the document]
Comment (jshawl) [reply src/ui/buttons/button.jsx]: I love it! This should make it easier to add in a unit test in the future if we need to. thanks! :shipit:

**Pull request merged**